contributors.json

```diff
@@ -147,6 +147,10 @@
     "country": "Nepal",
     "name": "Rajiv Sah"
   },
+  "raptor-dev": {
```

Comment:
This should be your GitHub username. Please change it to the exact Github user name